cmd/kubelet/app/server.go

```diff
@@ -166,7 +167,7 @@ func NewKubeletServer() *KubeletServer {
 		HealthzPort:                 10248,
 		HealthzBindAddress:          util.IP(net.ParseIP("127.0.0.1")),
 		RegisterNode:                true, // will be ignored if no apiserver is configured
-		OOMScoreAdj:                 -900,
+		OOMScoreAdj:                 -1000,
```

Comment:
I don't think the kubelet will have a big oom_score since its memory usage is pretty low when compared to user tasks. I'd expect that to be the case in the future as well. Setting it to a value lower than -900 would work, but might not be necessary in reality.